frontend/test/testSetup.js

```diff
@@ -33,6 +33,9 @@ loadAllRules(i18nMock)
 
 global.localVue = createLocalVue()
 
+// switch of warnings from bootstrap vue
+global.process.env.BOOTSTRAP_VUE_NO_WARN = true
```

Comment:
This is only for test cases!? Or do we need to have this in mind in case we get problems in the future?

Reply:
This is only for the test cases. We wiill have the same warnings when we use bootstrap vue's modals